### PR TITLE
yapf formatting and fixes

### PR DESCRIPTION
**Pull request by nicpottier**
For discussion.. run as:

```
% find . -name "*.py" | grep -v migrations | xargs yapf -i --style='{column_limit=119, indent_width: 4, coalesce_brackets: true, allow_split_before_dict_value: false, split_before_first_argument: false, dedent_closing_brackets: true }'
```

Some manual tweaks made to make flake8 pass

### CODE REVIEW

File: temba/api/models.py [hunk 1]
```diff
@@ -160,51 +164,42 @@ class WebHookEvent(SmartModel):
...
                     (TYPE_SMS_FAIL, "Outgoing SMS Failed to be Delivered to Recipient"),
-                    (ChannelEvent.TYPE_CALL_OUT, "Outgoing Call"),
-                    (ChannelEvent.TYPE_CALL_OUT_MISSED, "Missed Outgoing Call"),
-                    (ChannelEvent.TYPE_CALL_IN, "Incoming Call"),
-                    (ChannelEvent.TYPE_CALL_IN_MISSED, "Missed Incoming Call"),
-                    (TYPE_RELAYER_ALARM, "Channel Alarm"),
-                    (TYPE_FLOW, "Flow Step Reached"),
-                    (TYPE_CATEGORIZE, "Flow Categorization"))
+                    (ChannelEvent.TYPE_CALL_OUT,
+                     "Outgoing Call"), (ChannelEvent.TYPE_CALL_OUT_MISSED,
```
Comment (rowanseymour) [inline temba/api/models.py]: This is a bit weird

Comment (nicpottier) [reply temba/api/models.py]: Ya, there are definitely some pathological cases. That's why I did this so we can chat how we feel about those.

I think in an ideal world we would like it so we can rerun yapf on the entire codebase at any time and just accept those changes. That requires us either to gate pieces where we feel its formatting is wrong or to fix yapf where we think it is being weird. (either by tweaking settings or changing yapf itself through a PR or whatever)

File: temba/airtime/models.py [hunk 2]
```diff
@@ -23,17 +23,17 @@ class AirtimeTransfer(SmartModel):
...
-                      (SUCCESS, "Success"),
-                      (FAILED, "Failed"))
+    STATUS_CHOICES = ((PENDING, "Pending"), (SUCCESS, "Success"), (FAILED, "Failed"))
 
     org = models.ForeignKey(Org, help_text="The organization that this airtime was triggered for")
 
-    status = models.CharField(max_length=1, choices=STATUS_CHOICES, default='P',
-                              help_text="The state this event is currently in")
+    status = models.CharField(
+        max_length=1, choices=STATUS_CHOICES, default='P', help_text="The state this event is currently in"
```
Comment (norkans7) [inline temba/airtime/models.py]: I like the one argument/item per line I think these also could be so

**Comment by nicpottier** [comment]
Closing, this is a bit too ugly, but we are going to adopt @dodobas style filters when they don't fit on a line and normal indents (not visual) for model field definitions

**Pull request closed without merging**